kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -25,389 +24,419 @@ def parse(s: str) -> list[T]:
     return parse
 
 
-def node_id_like(s: str) -> NodeIdLike:
-    try:
-        return int(s)
-    except ValueError:
-        return s
+class KEVMDisplayOptions(DisplayOptions):
+    sort_collections: bool
 
+    @staticmethod
+    def default() -> dict[str, Any]:
+        return {'sort_collections': False}
 
-class KEVMCLIArgs(KCLIArgs):
-    @cached_property
-    def target_args(self) -> ArgumentParser:
-        args = ArgumentParser(add_help=False)
-        args.add_argument('--target', choices=['llvm', 'haskell', 'haskell-standalone', 'foundry'])
-        return args
+    @staticmethod
+    def update_args(parser: ArgumentParser) -> None:
+        parser.add_argument(
+            '--sort-collections',
+            dest='sort_collections',
+            action='store_true',
+            help='Sort collections before outputting term.',
+        )
 
-    @cached_property
-    def k_args(self) -> ArgumentParser:
-        args = super().definition_args
-        args.add_argument('--depth', default=None, type=int, help='Maximum depth to execute to.')
-        return args
 
-    @cached_property
-    def kprove_args(self) -> ArgumentParser:
-        args = ArgumentParser(add_help=False)
-        args.add_argument(
+class KOptions(KDefinitionOptions):
+    depth: int | None
+
+    @staticmethod
+    def default() -> dict[str, Any]:
+        return {'depth': None}
```

Comment:
This appears to just delete the defaults of the superclass `KDefinitionOptions`. Should we be doing something like:

```
def default() -> dict[str, Any]:
    return chain([super().default(), {'depth': None}])
```

Reply:
Ah, I see that `Options` uses `type(...).mro()` to build up the overall defaults...